components/service/glean/src/test/java/mozilla/components/service/glean/private/TimespanMetricTypeTest.kt

```diff
@@ -144,4 +144,25 @@ class TimespanMetricTypeTest {
         assertTrue(metric.testGetValue("store2") >= 0)
         assertEquals(1, testGetNumRecordedErrors(metric, ErrorType.InvalidValue))
     }
+
+    @Test
+    fun `test setRawNanos`() {
```

Comment:
Can you add a separate test to check that when using both APIs things don't go wrong? Looks like it shouldn't, but I'd rather be sure about it.